heron/common/tests/cpp/network/switch_unittest.cpp

```diff
@@ -26,13 +26,14 @@
 #include "basics/basics.h"
 #include "errors/errors.h"
 #include "threads/threads.h"
+#include "threads/spcountdownlatch.h"
 #include "network/network.h"
 #include "basics/modinit.h"
 #include "errors/modinit.h"
 #include "threads/modinit.h"
 #include "network/modinit.h"
 
-static const sp_uint32 SERVER_PORT = 61000;
+static sp_uint32 SERVER_PORT;
```

Comment:
Do we need to need to have a global variable for saving the port? Can we add something in the class itself?